sys/freebsd/ksem.txt

```diff
@@ -0,0 +1,20 @@
+# Copyright 2020 syzkaller project authors. All rights reserved.
+# Use of this source code is governed by Apache 2 LICENSE that can be found in the LICENSE file.
+
+include <sys/types.h>
+include <sys/ksem.h>
```

Comment:
So these are not compiled into the kernel by default. Could you please add a second commit which adds `sem_load="YES"` to the loader.conf used by syz-ci? This is in pkg/build/freebsd.go.

Reply:
Yes, I will open another PR for that.

Reply:
[PR](https://github.com/google/syzkaller/pull/2310)

Reply:
Thanks, I merged it. Just a note for next time: it wasn't really necessary to use a separate PR, you could just push additional commits here.